pixl_dcmd/src/pixl_dcmd/main.py

```diff
@@ -81,344 +81,139 @@ def anonymise_dicom(dataset: Dataset) -> Dataset:
 
     logger.info("Anonymising received instance")
 
-    # Rip out overlays/
-    dataset = remove_overlays(dataset)
-    logger.info("Removed overlays")
-
     # Merge tag schemes
     tag_operations = load_tag_operations(project_config)
-    all_tags = merge_tag_schemes(tag_operations, manufacturer=dataset.Manufacturer)
+    tag_scheme = merge_tag_schemes(tag_operations, manufacturer=dataset.Manufacturer)
 
-    # Apply scheme to instance
-    dataset = apply_tag_scheme(dataset, all_tags)
-    # Apply whitelist
-    dataset = enforce_whitelist(dataset, all_tags)
+    modalities = project_config.project.modalities
 
     logger.info(
-        f"DICOM tag anonymisation applied according to {project_config.tag_operation_files}"
+        f"Applying DICOM tag anonymisation according to {project_config.tag_operation_files}"
     )
-    # Write anonymised instance to disk.
-    return dataset
+    logger.debug(f"Tag scheme: {tag_scheme}")
+
+    if (0x0008, 0x0060) in dataset and dataset.Modality not in modalities:
+        msg = f"Dropping DICOM Modality: {dataset.Modality}"
+        logger.error(msg)
+        raise ValueError(msg)
 
+    logger.warning("Anonymising received instance")
```

Comment:
Should be an info level, though I know we've done terrible things to make it visible from the orthanc plugin. 

Now that we've got loguru in core (used core, cli and pixl_imaging) we can do some simple logging config in the `pixl.py` plugin, the fun thing is that it only prints stdout not stderr, but I think this should get the orthanc-anon to print the logs as we'd expect. 

```
# Set up logging as main entry point
logger.remove()  # Remove all handlers added so far, including the default one.
logging_level = config("LOG_LEVEL")
if not logging_level:
    logging_level = "INFO"
logger.add(sys.stdout, level=logging_level)

logger.warning("Running logging at level {}", logging_level)

```